client/src/components/RecipeCard.js

```diff
@@ -1,11 +1,18 @@
 import React, { useState } from "react";
-
 import styled from "styled-components";
-
-export default function RecipeCard({ addRecipeToFavorites, recipe }) {
-  const [showText, setShowText] = useState(false);
-  const onClick = () => setShowText(!showText);
-  const Text = () => (
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faHeart } from "@fortawesome/free-solid-svg-icons";
+export default function RecipeCard({
```

Comment:
```suggestion
import { faHeart } from "@fortawesome/free-solid-svg-icons";


export default function RecipeCard({
```
Maybe add a linebreak here